CRM/Core/Error.php

```diff
@@ -598,6 +598,7 @@ public static function debug_log_message($message, $out = FALSE, $prefix = '', $
     $config = CRM_Core_Config::singleton();
 
     $file_log = self::createDebugLogger($prefix);
+    $file_log->setLocale(CRM_Core_I18n::getLocale());
```

Comment:
Shouldn't that be inside of `createDebugLogger()`? (*i.e. it's a factory-function that hides the details of how the log-object is instantiated*)